web/html/src/manager/systems/virtual-list.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from "react";
+import { useState, useEffect } from "react";
+import Network from "utils/network";
+import * as Systems from "components/systems";
+import { Utils } from "utils/functions";
+import { Table } from "components/table/Table";
+import { Column } from "components/table/Column";
+import SpaRenderer from "core/spa/spa-renderer";
+import { IconTag } from "components/icontag";
+import { SearchField } from "components/table/SearchField";
+
+// See java/code/src/com/suse/manager/webui/templates/systems/virtual-list.jade
+type Props = {
+  /** Locale of the help links */
+  docsLocale: string;
+  isAdmin: boolean;
+};
+
+function VirtualSystems(props: Props) {
+  const [selectedSystems, setSelectedSystems] = useState([]);
+  const [selectedSystemsCount, setCount] = useState(0);
+
+  const handleSelectedSystems = (data) => {
+    setSelectedSystems(data);
+  };
+
+  useEffect(()=> {
+    setCount(selectedSystems.length);
+    document.getElementById("header_selcount").innerHTML =
```

Comment:
What's the intent here? We don't want to manually manage HTML content like this.

Reply:
I think this it to update the SSM selection count at the top. Not sure how to cleanly do it